### ISSUE TITLE
Crash when used as part of SPM test target

### ISSUE THREAD

**Issue by BenStairway** [algolia/algoliasearch-client-swift#619]
**Describe the bug 🐛**
A force unwrap in `AbstractClient.defaultUserAgents()` causes a crash when the code is executed as part of a SPM test target ran on a Mac destination. The code assumes that the bundle's info.plist contains `CFBundleShortVersionString`. The test bundle does not contain this value.

**To Reproduce 🔍**
1. Create a swift package.
2. Add `AlgoliaSearch` as a dependencies to the target.
3. In the main target add a function that create an instance of `Client` and performs a network request.
4. Add a test to the test target that invokes the function created in step 3.
5. Run the tests on a Mac destination.

**Expected behavior 💭**
The test target executes and exits normally without crashing.

**Environment:**
- AlgoliaSearch 7.0.1
- Xcode 11.3 beta 1
- MacOS 10.14.6

**Additional context**
Here's the crashing code in context:
```
  private static func defaultUserAgents() -> [LibraryVersion] {
    // Add this library's version to the user agents.
    let version = Bundle(for: Client.self).infoDictionary!["CFBundleShortVersionString"] as! String
```
Based on the comment I suspect that this code isn't working as expected regardless of the crash. The code seems to be assuming that it is being executed as part of a dynamic library (which was the norm when Cocoapods reigned supreme). When the code is compiled into the main binary (SPM) then it will use the value from the main app's info.plist which is unlikely to be useful.

The reason to run the tests on a Mac destination instead of an iOS simulator is that the tests execute faster (because don't need to spin up the simulator).

**Comment by VladislavFitz** [comment]
Thank you for reporting this bug. 
The issue is solved in the branch `release/7.0.2`. You can use it in your project. 
We will make it a part of a future release.

**Comment by BenStairway** [comment]
Great, thanks for fixing!
